### DISCUSSION TITLE
Correct syntax to find status:inerror

### DISCUSSION THREAD

**Discussion by fearnie85** [gdbarron/VenafiTppPS#138] [Q&A]
https://venafitppps.readthedocs.io/en/latest/functions/Find-TppCertificate/#-inerror

Hi Team, 

I am quite new to this and wondering if someone can help me. I am currently just playing around and wanting to find how to pull tppcertificates that are in error, but I am unsure of how to do this and I am a little confused by the documentation.

What would be the input to pull ALL certs inerror?

For example, I know I have 4 of these. I have successfully done this using the invoke-restmethod, but I can not do this using the PS cmdlet. 

Thanks.

**Comment by gdbarron** [comment]
Hi @fearnie85.  This should do it, `Find-TppCertificate -Path '\ved\policy' -Recursive -InError $true`.

**Reply by gdbarron** [reply]
Please let me know where the confusion is with the documentation so I can clear it up.  Thanks!

**Reply by fearnie85** [reply]
perfect thanks. just a follow on question, how do I get more details regarding the error now that it has returned these errors? thanks.

**Reply by gdbarron** [reply]
`Find-TppCertificate -Path '\ved\policy' -Recursive -InError $true | Get-TppCertificateDetail`

**Reply by fearnie85** [reply]
perfect thanks! I think I need to read more in the examples. I usually learn from reading other peoples scripts, but I don't seem to see many on here.
Is there some slack group for example?

**Reply by gdbarron** [reply]
there are venafi slack groups, but I'm not aware of any specific to this module.
